libs/designer/src/lib/core/actions/bjsworkflow/serializer.ts

```diff
@@ -999,7 +999,7 @@ const serializeSubGraph = async (
 
   if (graphDetail?.inputs && graphDetail?.inputsLocation) {
     const inputs = serializeParametersFromManifest(getOperationInputsToSerialize(rootState, graphId), { properties: graphDetail } as any);
-    safeSetObjectPropertyValue(result, [...graphInputsLocation, ...graphDetail.inputsLocation], inputs, true);
+    safeSetObjectPropertyValue(result, [...graphInputsLocation, ...graphDetail.inputsLocation], inputs);
```

Comment:
no more "true" needed? How is this affecting the final serialized output? Do you have old vs new example?